### PR TITLE
Add names, locations and github usernames of all pod members

### LINKED ISSUE #13: Add the pod!
Currently, then repository has members of the MLH team on it, but you should replace them with members of the pod.

You can update information in `_data/fellows.yml`

You should work on:
- [ ] Replacing the images
- [ ] Update the names and locations of each fellow
- [ ] Update the pod number inside of `_config.yml`

### PR DESCRIPTION
**Pull request by jaygala223**
## Please, go through these steps before you submit a PR.

- [X] My Pod Leader knows I'm working on this Pull Request
- [X] I've explained what the Pull Request is adding.
- [X] I've explained why this is important.

Fixes #13

### CODE REVIEW

**Comment by jaygala223** [comment]
Hi @akshgpt7, can you please review this PR?

File: Gemfile.lock [hunk 1]
```diff
@@ -37,7 +37,7 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    liquid (4.0.3)
+    liquid (4.0.4)
```
Comment (akshgpt7) [inline Gemfile.lock]: Need to remove this file from the PR.

File: _config.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-title: "Pod 22.PX.X"
+title: "Pod 23.APR.1"
```
Comment (akshgpt7) [inline _config.yml]: This is a duplicate change from #15.
You should either have it here, or just over there.

**Review by akshgpt7 — COMMENTED** [review]
The changes look great! Love to see the fellows in there ;)
Can you just address the few comments and I'll merge it.

**Comment by akshgpt7** [comment]
Also, I don't see where you've uploaded the images and they're not showing up in the site either. can you let me know if you committed them?

File: _config.yml (hunk 2 above)
Comment (jaygala223) [reply _config.yml]: Yes. sorry, I have removed it from this PR

File: Gemfile.lock (hunk 1 above)
Comment (jaygala223) [reply Gemfile.lock]: Sorry for this. I have removed it.

**Comment by jaygala223** [comment]
> Also, I don't see where you've uploaded the images and they're not showing up in the site either. can you let me know if you committed them?

Actually, me and @ohsweetmani divided our tasks so we can work on them asynchronously... so I was working on updating the names and locations of the fellows and Priyanka will be updating the images of the fellow

**Comment by ohsweetmani** [comment]
Please review this pull request,  @akshgpt7

**Comment by akshgpt7** [comment]
@jaygala223 can you please fix the merge conflicts, will be ready to merge after that.

**Comment by jaygala223** [comment]
Hi @akshgpt7, I and @ohsweetmani have resolved the merge conflicts. Please check

**Comment by akshgpt7** [comment]
LGTM, Thanks a lot for picking this!

**Pull request merged**